NEWS.md

```diff
@@ -73,6 +73,7 @@
 * New linter `duplicate_argument_linter()` checks that there are no duplicate arguments supplied to
 function calls. (#850, #851, @renkun-ken)
 * Several optional `Imported` packages have become `Suggested` dependencies: `httr`, `testthat`, and `rstudioapi`. This should allow snappier CI builds for usages not relying on some more "peripheral" features of the package.
+* Error message for mismatched starts and ends of exclusion ranges now includes line numbers. (#860, @danielinteractive)
```

Comment:
I think you'll want to just merge this with the earlier NEWS item (still including the issue number & citing yourself of course)

Reply:
Can you help me which line number this would be? I stared at the list for 5 mins now and cannot see the matching item :-) thx!

Reply:
I can see that you did some changes 1 year ago in https://github.com/jimhester/lintr/pull/524 but that is part of previous NEWS already. So I would suggest to keep this as a separate bullet here since there is no other NEWS entry here which matches?

Reply:
sorry! just realized no item was added for #571 in #575. could you mention #571 and @AshesITR in an updated item?

Reply:
thx - done